react/package.json

```diff
@@ -24,7 +24,7 @@
         "@babel/preset-typescript": "^7.21.0",
         "@changey/react-leaflet-markercluster": "^4.0.0-rc1",
         "@reduxjs/toolkit": "^1.8.4",
-        "@tacc/core-styles": "^2.22.6",
+        "@tacc/core-styles": "^2.22.3",
```

Comment:
Why the downgrade? What did I add after 2.22.3 that smelled so bad? 😄 Latest release currently is 2.23.1.

Reply:
I'll bump to latest release. i think we were just matching TUP for no reason.

Reply:
Okay. Safety. Makes sense. Core-Styles follows [SemVer](https://semver.org/), so you may trust `X.`**`Y`**`.Z` and `X.Y.`**`Z`** to not break anything. _Example: I delay `2.24.0` release until I fix bugs on redesigned tables._